packages/clerk-js/src/v4/customizables/makeCustomizable.tsx

```diff
@@ -0,0 +1,27 @@
+import { useContainerMetadata } from '../elements/hooks';
+
+type Customizable<T = {}> = { appearanceKey?: string | string[] } & T;
+type CustomizablePrimitive<T> = React.FunctionComponent<Customizable<T>>;
+
+const fakeGenerateSemanticClassName = (props: any) => {
```

Comment:
❓ Why is this called `fakeGenerateSemanticClassName`? In other words, what's fake about it?

Reply:
(already discussed offline, posting here for the sake of completeness and resolving this one  :) )

`fakeGenerateSemanticClassName` was a placeholder for the actual className generation helpers which are now merged in this PR